Editor.php

```diff
@@ -2047,7 +2048,7 @@ private function _ssp_limit($query, $http)
 	private function _get_where($query)
 	{
 		for ($i = 0; $i < count($this->_where); ++$i) {
-			if (is_callable($this->_where[$i])) {
+			if ($this->_where[$i] instanceof \Closure) {
```

Comment:
the reason is `Query::where` does support only `\Closure`, so no BC break, actually it is a bugfix, as callable looking array might be regular where